locales/en/default.json

```diff
@@ -159,6 +159,13 @@
         "pageTitle": "User Activity Report results"
       }
     },
+    "user-management": {
+      "serviceName": "Administration",
+      "organisationsPage": {
+        "title": "Manage organisations",
```

Comment:
Is it possible to use placeholder variables here so this could be "Manage organisation: ACME inc"?

Reply:
Done one better - variable comes from orgInfo retrieved, so is dynamic.